longshot/application/main.tf

```diff
@@ -81,6 +84,11 @@ variable "database_size_gb" {
   default     = 100
 }
 
+variable "with_newrelic" {
```

Comment:
Thought from something in you mentioned in another PR related to this issue the other day. Is this module maybe better put in `shared` module, if we want to use it across properties? Or are we not sure we want to stick with New Relic, and this is our first test to see how we like it on Heroku?

Reply:
Sorry, which module are you referring to here? The overall `longshot/application` module?

Reply:
Chatted this through IRL. For anyone else who's following along – the `with_newrelic` variable here is just an argument for the module (think of it as a parameter you'd pass a function in other languages), so it's inherently tied to this module and isn't something we could reasonably extract.

We could definitely extract pieces of this `longshot/application` module into more reusable chunks for elsewhere (so say, bundling the `aws_ssm_parameter` & `heroku_addon` into a `shared/heroku-newrelic` module that exports the expected config values) and I'm planning on tackling that further in #28 after this app is wrapped up.